scripts/core/memory.yml

```diff
@@ -540,3 +540,93 @@ type: class
 desc: "C++ wrapper for memory allocation"
 name: $xMem
 owner: $xContext
+--- #--------------------------------------------------------------------------
+type: enum
+desc: "atomic access attribute flags"
+class: $xMem
+name: $x_memory_atomic_attr_exp_flags_t
+version: "1.7"
+etors:
+    - name: NO_ATOMICS
+      desc: "No atomics performed on the pointer"
+    - name: NO_HOST_ATOMICS
+      desc: "Atomics are not performed on the pointer by the host"
```

Comment:
can we use allowed instead of performed?

Reply:
@MichalMrozek : yes. Please check.